### ISSUE TITLE
Inconsistency in GSM8K Baseline Performance for Llama-3-8B

### ISSUE THREAD

**Issue by Dominic789654** [Julia-LiuJ/NLFT#1]
I've noticed a potential discrepancy in the reported baseline performance of Llama-3-8B with Chain-of-Thought (CoT) prompting on GSM8K:

1. In Figure 4/5, at x=0 (baseline), the model shows only 9% accuracy on GSM8K
<img width="620" alt="image" src="https://github.com/user-attachments/assets/38c41c4c-5b88-4530-81a1-a23bdeb3dab9" />

2. However, this seems inconsistent with:
   - My personal experiments where Llama-3-8B with CoT achieves ~50% accuracy
   - Figure 9 of your paper, where the Reft method baseline (Llama-3-8B with CoT) shows around 50% accuracy

Could you please clarify this apparent inconsistency in the baseline performance? Is there perhaps a difference in the experimental setup or evaluation methodology that explains these varying results?

**Comment by WJMacro** [comment]
I also find the baseline seems to be unusually weak. After trying to reproduce the result, I think the performanc gain may mainly comes from the fact that NLFT actually distills the COT result of Llama-3-8B-Instruct and SFT does not. (see issue #4)

**Comment by lrlbbzl** [comment]
I agree with you... I previously achieved about 56% accuracy using Llama-3-8B with CoT.

**Comment by WJMacro** [comment]
> I agree with you... I previously achieved about 56% accuracy using Llama-3-8B with CoT.

I think they only use COT to fix the format of the final output, as they only use output examples in the prompt in Appendix.
